src/game/client/c_env_projectedtexture.cpp

```diff
@@ -407,11 +406,7 @@ void C_EnvProjectedTexture::UpdateLight( void )
 	}
 
 	g_pClientShadowMgr->SetFlashlightLightWorld( m_LightHandle, m_bLightWorld );
-
-	if ( !asw_perf_wtf.GetBool() && !m_bForceUpdate )
```

Comment:
Did you mean to remove the `!m_bForceUpdate` condition as well?

Reply:
Re-read it a few times, we do need to respect that flag. Double-checking against Enhanced Source, [they lopped it off too](https://github.com/Enhanced-Source-Project/Enhanced-Source/commit/447448433789d3ed2b7abbf807e303d9c6271ca7#diff-af1bfa88672f0fb1f774c1f96c97b4f2d00bf6fd8f4eff5f3636d43c407b0b5dL418-L422). Whoops.